test-it/ViaTests/ArrayDemo.via

```diff
@@ -10,6 +10,7 @@ define(ArrayDemo dv(.VirtualInstrument (
 
     e(a(.Int32 * *)   variableArray2d)
     e(a(.Int32 2 3)   fixedArray2d)
+    e(a(.Int32 0 0)   fixedArray2dEmpty)
```

Comment:
Maybe add comments to via tests with items that appear to be unused like this one to note that they are read programmatically by a karma test.